### PR TITLE
Feature/installer ui options

### PR DESCRIPTION
**Pull request by jofsky**
Made changes to metadeploy installer to make human readable step names for metadataETL tasks.
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by juliesnow — DISMISSED** [review]
approve changes made

**Review by davidmreed — APPROVED** [review]

File: cumulusci.yml [hunk 1]
```diff
@@ -288,14 +288,24 @@ flows:
         steps:
             1: 
                 task: add_application_type_picklist_values
+                ui_options:
+                    name: EDA - Application Type Picklist Values
```
Comment (jofsky) [inline cumulusci.yml:292]: @MatthewBlanski @lifewithryan @jstvz  whats your take on EDA vs K12 for the name?

Comment (MatthewBlanski) [reply cumulusci.yml:292]: I am all right with it in K-12 since it is explicit. You may want to check with @JasonFitzy

Comment (jstvz) [reply cumulusci.yml:292]: I think we're using the `EDA -` prefix to denote EDA's pre/post-install steps. I think replacing the prefix with 'Add' may be clearer.

**Review by jstvz — APPROVED** [review]

**Comment by JasonFitzy** [comment]
Thanks @MatthewBlanski  I agree and it looks fine.

**Pull request merged**